docs/tutorials/services/navigate-with-rover-base.md

```diff
@@ -0,0 +1,646 @@
+
+   {{<imgproc src="/tutorials/navigate-with-rover-base/leo-in-office.png" resize="500x" declaredimensions=true alt="Leo rover that is navigating using the navigation service in a robotics lab.">}}
+
+2. **A movement sensor with GPS position and angular and linear velocity readings**
+
+   We used three movement sensors to satisfy these requirements:
+
+   1. A [SparkFun GPS-RTK-SMA Breakout](https://www.sparkfun.com/products/16481) [movement sensor](/components/movement-sensor/) configured as a [`gps-nmea-rtk-serial`](/components/movement-sensor/gps/gps-nmea-rtk-serial/) model, providing GPS position measurements.
+   2. A [`wheeled-odometry`](/components/movement-sensor/wheeled-odometry/) model gathering angular and linear velocity information from the [encoders](/components/encoder/) wired to our base's [motors](/components/motor/).
+   3. A [`merged`](/components/movement-sensor/merged/) model aggregating the readings together for the navigation service to consume.
+
+{{% alert title="Tip" color="tip" %}}
+
+If you are using different hardware than us, the navigation setup process will be mostly the same, but you will need to substitute your robot's components.
+
+{{% /alert %}}
+
+Before you start, make sure to create a robot in [the Viam app](https://app.viam.com) and [install `viam-server`](/installation/) on your robot.
+
+## Configure the components you need
+
+First, configure the components of your robot.
+If you are using different _models_ of hardware, adjust your configuration accordingly.
+
+### Configure a board with `"digital_interrupts"`
+
+First, configure the [board](/components/board/) local to your rover.
+Follow [these instructions](/components/board/#configuration) to configure your board model.
+We used a [`jetson` board](/components/board/jetson/), but you can use whatever model of board you have on hand, as the [resource's API](/components/board/#api) is hardware agnostic.
+
+Configure a board named `local` as shown below:
+
+![Configuration of a jetson board with digital interrupts in the Viam app config builder.](/tutorials/navigate-with-rover-base/board-config-builder.png)
+
+Configure [digital interrupts](/components/board/#digital_interrupts) on your board to signal precise GPIO state changes to the [encoders](/components/encoder/) on your rover base.
+Copy and paste the following into your board's **Attributes** to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "digital_interrupts": [
+    {
+      "name": "ra",
+      "pin": "31"
+    },
+    {
+      "pin": "29",
+      "name": "rb"
+    },
+    {
+      "pin": "23",
+      "name": "lb"
+    },
+    {
+      "name": "la",
+      "pin": "21"
+    }
+  ]
+}
+```
+
+Save your config.
+
+### Configure a rover base with encoded motors
+
+Now, configure your rover base to act as the moving platform of the navigating robot.
```

Comment:
Let's remove some of the ones that aren't really needed.
```suggestion
Configure your rover base to act as the moving platform of the navigating robot.
```

Reply:
Ok, sounds good. Will try to keep this in mind